dockers/docker-ptf/Dockerfile.j2

```diff
@@ -12,7 +12,7 @@ FROM {{ prefix }}debian:buster
 USER root
 WORKDIR /root
 
-MAINTAINER Pavel Shirshov
+LABEL maintainer="Pavel Shirshov"
```

Comment:
Pavel is no longer working on SONiC.

Reply:
I have removed the LABEL maintainer for now. We can probably add generic label at a later date.